src/Token.cpp

```diff
@@ -0,0 +1,116 @@
+#include <Logger.hpp>
+#include <Token.hpp>
+
+#include <sstream>
+#include <vector>
+
+Token::Token(TokenType type, const std::string &string)
+	: _type(type), _string(string)
+{
+}
+
+Token::~Token()
+{
+}
+
+Token::Token(const Token &src) : _type(src._type), _string(src._string)
+{
+}
+
+const std::string &Token::getString() const
+{
+	return (_string);
+}
+
+TokenType Token::getType() const
+{
+	return (_type);
+}
+
+#ifndef TOKENLIST_SPECIAL_CHAR
+#define TOKENLIST_SPECIAL_CHAR "{};"
+#endif
+
+TokenType identifyToken(std::string str)
+{
+	if (str == "}")
+		return (TokenType::CLOSE_BRACKET);
+	else if (str == "{")
+		return (TokenType::OPEN_BRACKET);
+	else if (str == ";")
+		return (TokenType::SEMICOLON);
+	else
+		return (TokenType::WORD);
+}
+
+void stringToLower(std::string &str)
+{
+	for (size_t i = 0; i != str.size(); i++)
+		str.at(i) = std::tolower(str.at(i));
+}
+
+bool SkipComments(std::stringstream &stream, std::string &str)
+{
+	size_t pos = str.find('#');
+	if (pos != std::string::npos)
+	{
+		std::string filler;
+		str.erase(pos);
+		getline(stream, filler);
+	}
+	return (str.empty());
+}
+
+void splitString(std::string input, std::vector<Token> &list)
```

Comment:
should input be a reference? Not sure, but I think that input could be const &

Reply:
No it can't be const since i'm editing the input string. 
a reference, could be, but it's not required tbh;